docs/gno-tooling/cli/gnokey/querying-a-network.md

```diff
@@ -0,0 +1,199 @@
+a specific remote address from which the state will be retrieved.
+
+To send ABCI queries, you can use the `gnokey query` subcommand, and provide it
+with the appropriate query. The `query` subcommand allows us to send different
+types of queries to a gno.land network.
+
+Below is a list of queries a user can make with `gnokey`:
+- `auth/accounts/{ADDRESS}` - returns information about an account
+- `bank/balances/{ADDRESS}` - returns balances of an account
+- `vm/qfuncs` - returns the exported functions for a given pkgpath
+- `vm/qfile` - returns the list of files for a given pkgpath
+- `vm/qeval` - evaluates an expression in read-only mode on and returns the results
+- `vm/qrender` - shorthand for evaluating `vm/qeval Render("")` for a given pkgpath
+
+Let's see how we can use them.
+
+## `auth/accounts`
+
+We can obtain information about a specific address using this subquery. To call it,
+we can run the following command:
+
+```bash
+gnokey query auth/accounts/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5 -remote https://rpc.gno.land:443
+```
+
+With this, we are asking the Portal Loop network to deliver information about the
+specified address. If everything went correctly, we should get the following
+output:
+
+```bash
+height: 0
+data: {
+  "BaseAccount": {
+    "address": "g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5",
+    "coins": "227984898927ugnot",
+    "public_key": {
+      "@type": "/tm.PubKeySecp256k1",
+      "value": "A+FhNtsXHjLfSJk1lB8FbiL4mGPjc50Kt81J7EKDnJ2y"
+    },
+    "account_number": "0",
+    "sequence": "12"
+  }
+}
+```
+
+The return data will contain the following fields:
+- `height` - the height at which the query was executed. This is currently not
+  supported and is `0` by default.
+- `data` - contains the result of the query.
+
+The `data` field returns a `BaseAccount`, which is the main struct used in [TM2](../../../concepts/tendermint2.md)
+to hold account data. It contains the following information:
+- `address` - the address of the account
+- `coins` - the list of coins the account owns
+- `public_key` - the TM2 public key of the account, from which the address is derived
+- `account_number` - a unique identifier for the account on the gno.land chain
+- `sequence` - a nonce, used for protection against replay attacks
+
+## `bank/balances`
+
+With this query, we can fetch [coin](../../../concepts/stdlibs/coin.md) balances
+of a specific account. To call it, we can run the following command:
+
+```bash
+gnokey query bank/balances/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5 -remote https://rpc.gno.land:443
+```
+
+If everything went correctly, we should get an output similar to the following:
+
+```bash
+height: 0
+data: "227984898927ugnot"
+```
+
+The data field will contain the coins the address owns.
+
+## `vm/qfuncs`
+
+Using the `vm/qfuncs` query, we can fetch exported functions from a specific package
+path. To specify the path we want to query, we can use the `-data` flag:
```

Comment:
Maybe we could add a link to a section that explains the functions exported by a package or a realm?

Reply:
This is specific to Go, I think most people who are reading this will understand that it's functions starting with a capital letter. Not sure where to link to tbh

Reply:
Yes I actually meant to link to a resource that explains in a few words the principle of Gno exported functions that can be called using a Gno client. It’s true that you mention packages in the line I'm commenting on, but right below that, your command lists the functions of a realm.

So it might make sense to provide some context by pointing to [that section](https://docs.gno.land/how-to-guides/simple-contract#2-start-writing-code) of the docs for example. 
What do you think?

Reply:
Following internal discussions I'm resolving this convo